utilities/plotting/plotBayes.m

```diff
@@ -3,13 +3,13 @@ function plotBayes(problem, results)
     figure(10); clf; plotRefSLD(problem,results)
     
     figure(30); clf;
-    bayesShadedPlot(problem,results,'fit','average','KeepAxes',true,'interval',65,'q4',false)
+    bayesShadedPlot(problem,results,'fit','average','KeepAxes',true,'interval',95,'q4',false)
```

Comment:
This value went from 95 to 65 and now back to 95, it might be better to provide these as options to the function